packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -821,6 +836,49 @@ class Auth {
         return token
     }
 
+    /**
+     * A wrapper method for commerce-sdk-isomorphic helper: authorizePasswordless.
+     */
+    async authorizePasswordless(parameters: AuthorizePasswordlessParams) {
+        const userid = parameters.userid
+        const mode = this.passwordlessConfig.mode === 'sms' ? 'sms' : 'callback'
+        const callbackURI = this.passwordlessConfig.callbackURI
+
+        await helpers.authorizePasswordless(
+            this.client,
+            {
+                clientSecret: this.clientSecret
+            },
+            {
+                ...(callbackURI && {callbackURI: callbackURI}),
+                userid,
+                mode: mode
+            }
+        )
+    }
+
+    /**
+     * A wrapper method for commerce-sdk-isomorphic helper: getPasswordLessAccessToken.
+     */
+    async getPasswordLessAccessToken(parameters: LoginPasswordlessParams) {
+        const pwdlessLoginToken = parameters.pwdlessLoginToken
+        const token = await helpers.getPasswordLessAccessToken(
+            this.client,
+            {
+                clientSecret: this.clientSecret
+            },
+            {
+                pwdlessLoginToken
+            }
+        )
+        const isGuest = false
+        this.handleTokenResponse(token, isGuest)
+        if (onClient()) {
+            void this.clearECOMSession()
+        }
```

Comment:
Is there any issue with the order of these 2 calls? I'm not sure what `clearECOMSession` does, but maybe it clears some kind of local storage.. and maybe handleTokenResponse does too. Is there any concern that we might clear values that we just wrote? or that there is a race condition here?

Reply:
In my previous PR @kevinxh mentioned that this started being necessary to call after all on success logins, but we are missing some documentation on why this is needed.

@shethj I see you authored this `clearECOMSession()` I was wondering if you could give more context?